src/pytest_elasticsearch/factories.py

```diff
@@ -83,6 +84,7 @@ def elasticsearch_proc_fixture(request):
         """Elasticsearch process starting fixture."""
         config = return_config(request)
         elasticsearch_host = host or config['host']
+        executable = config['executable'] or executable
```

Comment:
try the other way around

Reply:
In that case we should remove the default value inside the function ? Currently I think that if it is 
```
  executable = executable or config['executable'] 
```

it will never go to ```config['executable']``` since we assign default value to executable ?

Reply:
Oh, right, then the default value has to be provided by config (defined in plugin) and executable arg has to be set to None. Right now the parameter value won't ever be used.